Demos/Raven.NET.Demo.WebApi/Repositories/OrderRespository.cs

```diff
@@ -0,0 +1,58 @@
+using Raven.NET.Demo.WebApi.Model;
+using Raven.NET.Demo.WebApi.Repositories.Interfaces;
+using Raven.NET.Demo.WebApi.Services.Interfaces;
+
+namespace Raven.NET.Demo.WebApi.Repositories
+{
+    public class OrderRepository : IOrderRepository
+    {
+        private readonly Dictionary<Guid, Order> _orders = new();
```

Comment:
I've got some exceptions of watchers trying to access same collection simultaneously, might be worth to introduce concurrency. Wasn't able to replicate - treat as suggestion

Reply:
Done